Apollo.xcodeproj/project.pbxproj

```diff
@@ -963,6 +964,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				E6D21E1B2705016000110BBD /* Nimble in Frameworks */,
```

Comment:
This was missing from the `ApolloCodegenTests` causing build errors.

Reply:
Are we actually using nimble now?

Reply:
It's not the whole Quick framework and I do think it's an easier to read and understand matcher language. So yes, I'm good with us bringing it in and beginning to use it.

Is there any reason you particularly don't like it?

Reply:
No i was wondering because it's been linked due to transient dependencies for a while